runtime/environment-cpu.yml

```diff
@@ -38,3 +38,8 @@ dependencies:
       - git+https://github.com/ibarrond/Pyfhel.git@0dee7a77469627c4de4f2dca97cdc68bf006a77d
       - tensorflow_privacy==0.7.3
       - tensorflow-probability==0.15.0
+      - deepwalk==1.0.3
+      - imblearn==0.0
+      - gensim==3.8.3
+      - tensorboard-logger==0.1.0
```

Comment:
When your submission runs on the hosted infrastructure, you won't be able to get any of the tensorboard events out from that run, so I don't think this will be very useful.

Would it be the case that you could use this for only local experimentation in a local image where you've added this? Or is use of it heavily integrated throughout your code?